packages/manager/modules/bm-server-components/src/general-info-tile/index.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
```

Comment:
You should have a `@ovh-ux/ui-kit` import here, and pass it as dep of the `angular.module(...)`